test/test_da.py

```diff
@@ -6,6 +6,57 @@
 
 import numpy as np
 import ot
+from numpy.testing.utils import assert_allclose, assert_equal
+from ot.datasets import get_data_classif
+from ot.utils import unif
+
+np.random.seed(42)
+
+
+def test_sinkhorn_transport():
```

Comment:
thank you for the reply, it's good for me. I will modify the name of the test function